qoqo/tests/integration/noise_models/overrotation.rs

```diff
@@ -0,0 +1,231 @@
+// Copyright © 2021-2022 HQS Quantum Simulations GmbH. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the
+// License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+use pyo3::prelude::*;
+use qoqo::noise_models::*;
+#[cfg(feature = "json_schema")]
+use roqoqo::{noise_models::SingleQubitOverrotationOnGate, ROQOQO_VERSION};
+
+/// Test copy
+#[test]
+fn test_pyo3_init() {
+    pyo3::prepare_freethreaded_python();
+    Python::with_gil(|py| {
+        let br_type = py.get_type::<SingleQubitOverrotationOnGateWrapper>();
+        let br = br_type
+            .call0()
+            .unwrap()
+            .downcast::<PyCell<SingleQubitOverrotationOnGateWrapper>>()
+            .unwrap();
+        let br_copied = br
+            .call_method0("__copy__")
+            .unwrap()
+            .extract::<SingleQubitOverrotationOnGateWrapper>()
+            .unwrap();
+        let br_wrapper = br
+            .extract::<SingleQubitOverrotationOnGateWrapper>()
+            .unwrap();
+        assert_eq!(br_copied, br_wrapper);
+    })
+}
+
+/// Test debug
+#[test]
+fn test_pyo3_debug() {
+    pyo3::prepare_freethreaded_python();
+    Python::with_gil(|py| {
+        let br_type = py.get_type::<SingleQubitOverrotationOnGateWrapper>();
+        let br = br_type
+            .call0()
+            .unwrap()
+            .downcast::<PyCell<SingleQubitOverrotationOnGateWrapper>>()
+            .unwrap();
+        let br_wrapper = br
+            .extract::<SingleQubitOverrotationOnGateWrapper>()
+            .unwrap();
+
+        let br_clone = br_wrapper.clone();
+        assert_eq!(format!("{:?}", br_wrapper), format!("{:?}", br_clone));
+
+        let debug_string = format!("{:?}", br_clone);
+        assert_eq!(format!("{:?}", br_wrapper), debug_string);
+    })
+}
```

Comment:
This test isn't really doing anything, I would suggest adding a test for (in pure Rust) the clone, debug and partialeq of the Wrapper (see the operations folder). (For both wrapper structs)